### PR TITLE
Add InputBox for user defined number of bins

### LINKED ISSUE #74: Allow for custom bin-size in the histogram

### PR DESCRIPTION
**Pull request by xjules**
Resolves #74

- Instead of `bin_size` it makes sense to specify the number of bins.
- The default `bin_size` was adjusted to match the old ert-gui.

### CODE REVIEW

File: ertviz/models/plot_model.py [hunk 1]
```diff
@@ -146,10 +155,9 @@ def repr(self):
                 ]
             )
 
-        bin_count = int(math.ceil(math.sqrt(len(data[0]))))
         _max = max(map(max, data))
         _min = min(map(min, data))
-        bin_size = float((_max - _min) / bin_count)
+        bin_size = float((_max - _min) / self.bin_count)
```
Comment (oysteoh) [inline ertviz/models/plot_model.py:160]: self._bin_count i think it is called?

File: ertviz/views/paremeter_view.py [hunk 2]
```diff
@@ -47,6 +47,14 @@ def parameter_view(parent):
                             ],
                             value=["hist", "kde"],
                         ),
+                        html.Label("#Bins:"),
```
Comment (oysteoh) [inline ertviz/views/paremeter_view.py:50]: If available space - would it be nicer with `number of bins` ?

File: ertviz/models/plot_model.py (hunk 1 above)
Comment (xjules) [reply ertviz/models/plot_model.py:249]: `self.bin_count` is a property, which recomputes `bin_count` if necessary

File: ertviz/views/paremeter_view.py (hunk 2 above)
Comment (xjules) [reply ertviz/views/paremeter_view.py:50]: Wanted to save space, will check if the plot shrinks

File: ertviz/models/plot_model.py (hunk 1 above)
Comment (oysteoh) [reply ertviz/models/plot_model.py:249]: yes - of course... my mistake 😬

File: ertviz/views/paremeter_view.py (hunk 2 above)
Comment (xjules) [reply ertviz/views/paremeter_view.py:50]: The plot didn't shrink, so changed.

**Review by oysteoh — APPROVED** [review]
Please squash before merging! 🚀

**Pull request merged**